internal/cmd/gocdk/apply_test.go

```diff
@@ -33,10 +32,6 @@ func TestApply(t *testing.T) {
 	// terraform never previously applied
 	// terraform apply repeat
 
-	if _, err := exec.LookPath("terraform"); err != nil {
```

Comment:
Why remove the skip? If we're running on a dev's machine that doesn't have Terraform installed, then `go test ./...` should still succeed IMO.

Reply:
Sure, OK.